src/styles/applications/_closings.scss

```diff
@@ -0,0 +1,6 @@
+.weather_details_overflow {
```

Comment:
What part of the weather page does this relate to? Checking it out on dev. @tmccoy529